src/app/coursegrab/utils/push_notifications.py

```diff
@@ -26,13 +27,15 @@ def notify_users(section):
 
     android_tokens = []
     ios_tokens = []
+    # emails = []
 
     for user in users:
         if user.notification == ANDROID:
-            android_tokens.append(user.device_token)
-        elif user.notification == IPHONE:
-            ios_tokens.append(user.device_token)
-
+            android_tokens.extend(get_user_device_tokens(user.id, ANDROID))
+        elif user.notification == IOS:
+            ios_tokens.extend(get_user_device_tokens(user.id, IOS))
+        # elif user.notification == EMAIL:
+        #     emails.append(user.e)
```

Comment:
consider creating a github issue for implementing email and then tagging these code comments with the issue # so it's easy to find where you started implementing